.github/workflows/sync.yml

```diff
@@ -1,24 +0,0 @@
-name: Synchronize main with SVN repository
-on:
-  push: # Merged PR always creates push event.
-    branches: [ main ]
-jobs:
-  sync:
-    name: Run synchronizer
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-      - name: WordPress.org plugin asset/readme update
-        uses: 10up/action-wordpress-plugin-asset-update@stable # https://github.com/10up/action-wordpress-plugin-asset-update
-        env:
-          IGNORE_OTHER_FILES: true # Otherwise, workflow will fail for mass changes.
```

Comment:
Vybavuji si, že původní běh scriptu umíral. Myslím si, že to bylo proto, že tento script očekával pouze změnu v readme.txt a assetech. Pokud změn bylo více, tak script umřel a readme.txt se neaktualizovalo. Proměnná IGNORE_OTHER_FILES zajistí, že script neumře když změn je více.

A také připomínám, že WP realeasuje podle stable tagu v readme.txt. Ale my chceme releasovat na WP při vytvoření releasu v Githubu.

Chceme pořád workflow update-readme.yml ? Pokud ano, nechceme přidat proměnou IGNORE_OTHER_FILES ?